docs/parcels.md

```diff
@@ -0,0 +1,109 @@
+# Parcels
+
+A single-spa parcel is a bundle of functionality (like an application or component) meant to be controlled imperatively 
+by an application, Parcels use similar methodology as Applications, and can be considered an advanced single spa feature so we recommend [starting with Applications.](/docs/applications.md)
+A parcel can be as large as an application or as small as an component and written in 
+any language as long as it exports the correct lifecycle events. In a single-spa world, your SPA contains 
+many registered applications and potentially many parcels. Usually you'll want to mount a parcel within the 
+context of an application, this will automatically unmount the parcel when the application is unmounted.
+
+## Parcel Lifecycles
+To read more about lifecycles and how single-spa handles the concept start with [applications](/docs/applications.md#registered-application-lifecycle)
+
+### Bootstrap
+This lifecycle function will be called once, right before the parcel is
+mounted for the first time.
+
+```js
+function bootstrap(props) {
+  return Promise
+    .resolve()
+    .then(() => {
+        // This is where you do one-time initialization
+        console.log('bootstrapped!')
+        });
+}
+```
+
+### Mount
+This lifecycle function will be called whenever the parcel is not mounted and `mountParcel` is called. When
+called, this function should create DOM elements, DOM event listeners, etc. to render content to the user.
+
+```js
+function mount(props) {
+  return Promise
+    .resolve()
+    .then(() => {
+        // This is where you tell a framework (e.g., React) to render some ui to the dom
+        console.log('mounted!')
+        });
+}
+```
+
+### Unmount
+This lifecycle function will be called whenever the parcel is mounted, and either A) `unmount()` is called 
+or B) the parent parcel or Application is unmounted. When called, this function should clean up all DOM elements, 
```

Comment:
"should clean up" or "will clean up" ? 

If it is "will" I would recommend that simple syntactic change.